pywatts/modules/wrappers/sklearn_wrapper.py

```diff
@@ -105,17 +105,19 @@ def transform(self, **kwargs: xr.DataArray) -> xr.DataArray:
         :return: the transformed output
         """
         x_np = self._dataset_to_sklearn_input(kwargs)
+        target_names = self.targets
 
         if isinstance(self.module, TransformerMixin):
```

Comment:
Please change this if statement to an elif statement and add before an if statement checking if self.module is a selector mixin. If this is the case repeat line 111 also in this if clause and set target_names=[].